GREEDY_WEB/ClientApp/components/UserLogin.tsx

```diff
@@ -0,0 +1,116 @@
+﻿import * as React from 'react';
+import { RouteComponentProps } from 'react-router';
+import { Button } from 'reactstrap';
+import DocumentTitle from 'react-document-title';
+import { LoginForm } from 'react-stormpath';
+import axios from 'axios';
+import { Link, NavLink } from 'react-router-dom';
+import Cookies from 'universal-cookie';
+import FacebookLogin from 'react-facebook-login';
+
+export class UserLogin extends React.Component<RouteComponentProps<{}>> {
+    constructor(props) {
+        super(props);
+        this.onFormSubmit = this.onFormSubmit.bind(this);
+    }
+
+    onFormSubmit(e, next) {
+        e.preventDefault();
+        var data = e.data;
+
+        if (data.username.length < 1) {
```

Comment:
we have to have 1 as a constant because if we cant to change it as 5 (for instance) for security purpose, then we will have to change it in GREEDY/Controllers/RegistrationController.cs too. also error message will be changed too